apps/coscrad-frontend-e2e/src/e2e/categories/category-tree.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,65 @@
+describe('Categories Tree (Tree of Knowledge) flow', () => {
+    const pageLabel = 'Tree of Knowledge';
+
+    const dummyCompositeIdPrefix = 'category/9b1deb4d-3b7d-4bad-9bdd-2b0d7b11000';
+
+    const rootNodeCompositeIdentifier = `${dummyCompositeIdPrefix}0`;
+
+    const firstNodeCompositeIdentifier = `${dummyCompositeIdPrefix}1`;
+
+    const nestedNodeCompositeIdentifier = `${dummyCompositeIdPrefix}7`;
+
+    const accordionTestId = `resources-for-${nestedNodeCompositeIdentifier}`;
+
+    const treeWalkNodeIds = [1, 2, 5, 7];
+
+    beforeEach(() => {
+        cy.visit('/TreeOfKnowledge');
+    });
+
+    describe('the tree of knowledge', () => {
+        it(`should display the label ${pageLabel}`, () => {
+            cy.contains(pageLabel);
+        });
+    });
+
+    describe('the tree root node', () => {
+        it('should open', () => {
+            cy.getByDataAttribute(rootNodeCompositeIdentifier).click();
```

Comment:
Can you add an additional assertion that convinces you that the tree node is now open?